joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/MavenDependencies.scala

```diff
@@ -42,7 +42,7 @@ object MavenDependencies {
         "The compile class path may be missing or partial.\n" +
         "Results will suffer from poor type information.\n" +
         "To fix this issue, please ensure that the below command can be executed successfully from the project root directory:\n" +
-        s"mvn $MavenCliOpts " + fetchArgs.mkString(" ") + "\n\n"
+        s"mvn $$MavenCliOpts " + fetchArgs.mkString(" ") + "\n\n"
```

Comment:
We want this to output `$MAVEN_CLI_OPTS`, so I think the correct fix would be:
```suggestion
        s"mvn $$$MavenCliOpts " + fetchArgs.mkString(" ") + "\n\n"
```